### DISCUSSION TITLE
HACS implementation

### DISCUSSION THREAD

**Discussion by abrouz** [icanos/hassio-plejd#243] [Ideas]
Any plans for making a HACS implementation?
I'm guessing I'm not the only one looking for an easier implementation in case you are running Home Assistant core or Supervised
Kind regards
Anders

**Comment by SweVictor** [comment]
No plans as far as I know. Out of curiosity, what makes HACS better than the current way? Personally, I see HACS as non-native things and HA plugins like this one as more native to the platform, but I'm guessing you have a different take...

**Reply by abrouz** [reply]
Add-ons aren't supported in case you are running Home Assistant in Core or Container mode.
HACS howevere works like a charm ;)

**Reply by SweVictor** [reply]
Alright, makes sense. Yeah, I guess you would need to run the code yourself or start the Docker container some other way.

Either way - this repo is open for contributions so sure, why not, if that's a good way forward. @icanos probably has an opinion and/or a reason for why it isn't a HACS repo in the first place.

**Reply by icanos** [reply]
Can't say that I do. I haven't really had the time lately to look into what is needed and how to publish the addon in HACS. Please feel free to help out get it there if it would benefit the larger community :)
